lib/utils/form/pin_form_element.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:cupcake/utils/config.dart';
+import 'package:cupcake/utils/form/abstract_form_element.dart';
+import 'package:cupcake/utils/form/abstract_value_outcome.dart';
+import 'package:cupcake/utils/secure_storage.dart';
+import 'package:cupcake/utils/form/default_validator.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:local_auth/local_auth.dart';
+
+class PinFormElement extends FormElement {
+  PinFormElement({
+    String initialText = "",
+    this.password = false,
+    this.validator = defaultFormValidator,
+    required this.valueOutcome,
+    this.onChanged,
+    this.onConfirm,
+    this.showNumboard = true,
+    required this.label,
+  }) : ctrl = TextEditingController(text: initialText);
+
+  Future<void> loadSecureStorageValue(VoidCallback callback) async {
+    if (ctrl.text.isNotEmpty) return;
+    if (!CupcakeConfig.instance.biometricEnabled) return;
```

Comment:
am I missing something? why if biometric is NOT enabled, we `return`?
if this function is just for initializing the biometric auth, then let's change this function name to be more meaningful

Reply:
this function is for automatically loading secure storage value that should be "typed" into the pin input form once the scan is successful